pkgs/development/python-modules/ifconfig-parser/default.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "ifconfig-parser";
+  version = "0.0.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "12n52zqwwc2k8si74qlgx2c33mc9glnkx23gn55k2zqdv892ylbc";
+  };
+
+  meta = with stdenv.lib; {
+    description = "Unsophisticated python package for parsing raw output of ifconfig.";
+    homepage = https://github.com/KnightWhoSayNi/ifconfig-parser;
```

Comment:
```suggestion
    homepage = "https://github.com/KnightWhoSayNi/ifconfig-parser";
```

Reply:
Is that the convention? I mostly followed the example of other phython packages and they didn't have these quotes in the homepage URL.

Reply:
it's part of a recent rfc which deprecated unquoted urls